src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs

```diff
@@ -274,25 +313,32 @@ public Size2D MaximumSize
             set => SetValue(ClippingModeProperty, value);
         }
 
-        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <summary>
+        /// Sets the size of a view for the width, the height and the depth.<br />
+        /// The views default depth is the minimum of width and height.<br />
```

Comment:
when is the default used? Size object always has depth assigned. Is the default really min(width, height)?

Reply:
Done. I removed the line explaining default depth.